sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeViewModel.kt

```diff
@@ -44,9 +44,13 @@ class HomeViewModel @Inject constructor(
                 .first()
 
             _isFirstTimeLaunch.emit(isFirstTimeLaunch)
```

Comment:
@AVI5HEK, It would be great if we handle the `scope` by `launchIn` like we do for `getModelsUseCase` above case. So we can get rid of calling `.first()`, Would you mind updating the existing logic 🙏  

```kotlin
        isFirstTimeLaunchPreferencesUseCase()
            .onEach { isFirstTimeLaunch ->
                _isFirstTimeLaunch.emit(isFirstTimeLaunch)
            }
            .flowOn(dispatchersProvider.io)
            .catch { e -> _error.emit(e) }
            .launchIn(viewModelScope)
```

Reply:
Updated in https://github.com/nimblehq/android-templates/pull/478/commits/d8c44c06ca24799ca04afcc1b59913dd1c4725d9